ccd-definition/CaseEventToFields/FeatureToggle-WAToggle-SettleDiscontinue-nonprod.json

```diff
@@ -46,5 +46,27 @@
     "PageShowCondition": "confirmOrderGivesPermission = \"DO_NOT_SHOW_IN_UI\"",
     "ShowSummaryChangeOption": "N",
     "Publish": "Y"
+  },
+  {
+    "CaseTypeID": "CIVIL${CCD_DEF_VERSION}",
+    "CaseEventID": "LIP_CLAIM_SETTLED",
+    "CaseFieldID": "featureToggleWA",
+    "PageDisplayOrder": 99,
+    "PageFieldDisplayOrder": 1,
+    "DisplayContext": "READONLY",
+    "PageID": "WorkAllocationIntegrationFields",
+    "ShowSummaryChangeOption": "N",
+    "Publish": "Y"
```

Comment:
does this need a page show condition/field show condition?

Reply:
looks like there's inconsistency that some fields have it and some don't but should we put one on just in case?